src/uct/tcp/tcp_ep.c

```diff
@@ -527,9 +535,13 @@ static ucs_status_t uct_tcp_ep_keepalive_enable(uct_tcp_ep_t *ep)
     }
 
     if (iface->config.keepalive.cnt != UCS_ULUNITS_AUTO) {
+        if ((iface->config.keepalive.cnt == UCS_ULUNITS_INF) ||
+            (iface->config.keepalive.cnt >= (unsigned long)keepalive_cnt)) {
```

Comment:
seems keepalive_cnt can be un initialized here?

Reply:
good catch, fixed